File: fixtures/lens-coverage-workspace/java-messaging/src/main/java/example/GcpPubSubClient.java
package example;

import com.google.pubsub.v1.TopicName;
import com.google.pubsub.v1.ProjectSubscriptionName;

public class GcpPubSubClient {
    public void wire() {
        TopicName topic = TopicName.of("my-project", "orders-topic");
        ProjectSubscriptionName sub = ProjectSubscriptionName.of("my-project", "orders-sub");
    }
}

File: fixtures/lens-coverage-workspace/js-messaging/src/gcp-pubsub-client.ts
import { PubSub } from '@google-cloud/pubsub';

const pubsub = new PubSub();

async function publish() {
    await pubsub.topic('orders-topic').publish(Buffer.from('hello'));
}

async function subscribe() {
    pubsub.subscription('orders-sub').on('message', (msg) => msg.ack());
}
